data/keys.keymap

```diff
@@ -0,0 +1,45 @@
+﻿0:37
```

Comment:
Wouldn't that load //Function as part of it?
As it is separated at : and then all text on the right gets used.

Would require a method to dispose all if //* is found?